src/AVKit/Enums.cs

```diff
@@ -24,4 +25,17 @@ public enum AVKitError : nint {
 		PictureInPictureStartFailed = -1001
 	}
 #endif
+
+#if TVOS
+	// this enum only exists for TVOS
+	[TV (10,1)]
+	public enum AVKitMetadataIdentifier : nint {
```

Comment:
Most (at least for new API) cases where `enum` should be used is when `NS_STRING_ENUM` is used in the headers.